frontend/src/index.tsx

```diff
@@ -2,11 +2,16 @@ import React from 'react';
 import ReactDOM from 'react-dom';
 import './index.css';
 import App from './App';
+import GlobalStyles from './components/util/GlobalStyles';
+import { BrowserRouter } from 'react-router-dom';
 
 
 ReactDOM.render(
   <React.StrictMode>
-    <App />
+    <BrowserRouter>
+      <GlobalStyles/>
```

Comment:
I think, it will be better to have `<GlobalStyles />` above router

Reply:
Okk